lib/modules/commentSortBy.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+
+import _ from 'lodash';
+import { $ } from '../vendor';
+import { Module } from '../core/module';
+import { rawLocale } from '../utils/localization';
+import {
+	string,
+	watchForRedditEvents,
+} from '../utils';
+
+export const module: Module<*> = new Module('commentSortBy');
+
+module.beforeLoad = () => {
+	watchForRedditEvents('postModTools', () => {
+		applyToPost();
+	});
+};
+
+module.moduleName = 'commentSortBy';
+module.category = 'commentsCategory';
+module.disabledByDefault = false;
+module.description = 'addCommentSortPrefMenuItemDesc';
+
+export class SortByPrefLink {
+	static defaultLinkElement = (e => () => e().cloneNode(true))(_.once(() => SortByPrefLink.buildTagElement()));
```

Comment:
This class is overkill for creating a little link. Can you keep `function buildTagElement` and delete the rest of this class and `afterLoad`?